src/wp-includes/capabilities.php

```diff
@@ -593,6 +593,18 @@ function map_meta_cap( $cap, $user_id, ...$args ) {
 		case 'manage_privacy_options':
 			$caps[] = is_multisite() ? 'manage_network' : 'manage_options';
 			break;
+		case 'view_site_health_check':
+			$caps['view_site_health_checks'];
+			if ( ! wp_site_health_allow_single_health_check( $user_id, $args[0] ) ) {
+				$caps[] = 'do_not_allow';
+			}
+			break;
+		case 'view_site_health_debug_info':
```

Comment:
I was trying to separate the site health checks, and the debug info entries, might be something wrong with my thought process here, but ideally not all debug info is always as applicable to a subsite entry (for example the directory size check, which currently runs with a multisite check, while it may be more sensible to remove the capability to view it).

And since they're a different area, it didn't make sense to use the `view_site_health_check` for this as well.